pydantic/_internal/_config.py

```diff
@@ -54,8 +54,8 @@ class ConfigWrapper:
     validate_assignment: bool
     arbitrary_types_allowed: bool
     from_attributes: bool
-    # whether to use the used alias (or first alias for "field required" errors) instead of field_names
-    # to construct error `loc`s, default True
+    # whether to use the actual key provided in the data(e.g. alias or first alias for "field required" errors) instead of field_names
```

Comment:
```suggestion
    # whether to use the actual key provided in the data (e.g. alias or first alias for "field required" errors) instead of field_names
```